tests/auth_tests/test_validators.py

```diff
@@ -145,6 +145,24 @@ def test_help_text(self):
         )
 
 
+class MinimumLengthValidatorCustomErrorTest(SimpleTestCase):
```

Comment:
Question for this class, and each new test class added in this fie:
Should we move the new test method into the test class that already exists for this validator (should we move this test in the `MinimumLengthValidatorTest` class)?

Reply:
I'd be fine with that - are there any other opinions?

Reply:
I would prefer this

Reply:
I'd like this to be done

Reply:
Oh, OK - I didn't realize you were saying you wanted me to move the new tests into the existing classes. I'll push a new commit.

Reply:
Ah sorry, you're right my comment wasn't clear - thank you